tests/test_plotImageSubtractionCutouts.py

```diff
@@ -241,6 +242,23 @@ def test_generate_image_multisize_cutouts_with_metadata(self):
             # size (in inches) and the dpi (default=100), plus borders.
             self.assertEqual((im.height, im.width), (576, 645))
 
+    def test_generate_image_and_save_as_numpy(self):
+        """A different cutout size: the resulting cutout image is the same
```

Comment:
This docstring seems like it was copied from another test--please update to describe the test we're doing here.

Reply:
Sorry, that was silly! Fixed it.